doc/FEATURES_RELEASE_2.md

```diff
@@ -53,11 +54,86 @@ must display useful error messages about such constructs to the user.
 and `clash2`.
 
+## Identify Sequential DDIs - Complete
+
+### Description
+Sequential DDIs arise because the PML contains a workflow in which the
+dispensing of one drug is always followed by the dispensing of another drug that
+has an interaction with the first. The system should identify these sequential
+DDIs and the enclosing `sequence` construct.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/ddis.pml` which contains a
+sequential DDI. Press `Submit`. The system should display the DDI information
+for the sequential DDI found in the analysis tab including its enclosing
+construct and the drugs involved.
+
+## Identify Parallel DDIs - Complete
+
+### Description
+Parallel DDIs arise because the PML contains interacting drugs in two different
+"arms" of a PML branch construct. The system should identify these parallel DDIs
+and the enclosing `branch` construct.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/parallel.pml` which contains
+a parallel DDI. Press `Submit`. The system should display the DDI information
+for the parallel DDI found in the analysis tab including its enclosing construct
+and the drugs involved.
+
+## Report Alternative non-DDIS - Complete
+
+### Description
+Alternative non-DDIs arise because the PML contains interacting drugs in
+different "arms" of a PML selection construct. These are not DDIs but have the
+potential to become DDIs if contained within an iteration construct. The system
+should identify these alternative non-DDIs and the enclosing `selection`
+construct. It should display them as a warning to signal potential DDIs if not
+carefully handled.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/alternative_non_ddi.pml` which
+contains an alternative non-DDI. Press `Submit`. The system should display the
+information for the alternative non-DDI found in the analysis tab including its
+enclosing construct and the drugs involved.
+
+## Report Repeated Alternative DDIs - Complete
+
+### Description
+Repeated alternative DDIs arise because the PML contains interacting drugs in
+different "arms" of a PML selection construct. However this selection occurs
+within an iteration which may lead to an interaction when the selection is
+repeated and another "arm" is chosen. The system should identify these repeated
+alternative DDIs and the enclosing `selection` and `iteration` constructs.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/repeated_alternative.pml`
+which contains a repeated alternative DDI. Press `Submit`. The system should
+display the information for the repeated alternative DDI found in the analysis
+tab including its enclosing construct and the drugs involved.
+
+## Specify a Delay - Complete
+
+### Description
+The system should provide a way to say, at any point in a sequential workflow,
+that the enactment of the pathway will pause/delay for a given time.
```

Comment:
Do you think we should include an example of what a delay looks like in our system? Similar to how what we did with `Specifying a drug in PML` for release 1.